core/queue/lua/done.lua

```diff
@@ -1,9 +1,10 @@
--- KEYS: [QueueName] [TaskGroup]
```

Comment:
These aren't keys... you're supposed to call scripts with `KEYS` and non-key arguments `ARGV` and by letting Redis know something is a key, it can correctly route things if sharding.